### PR TITLE
feat(kafka-setup): Adds SASL SSL support in kafka setup docker image

### PR DESCRIPTION
**Pull request by pedro93**
Adds support for Kafka SASL SSL support in kafka-setup docker image.

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by RyanHolstien — APPROVED** [review]

**Review by anshbansal — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -61,8 +61,8 @@ project.ext.externalDependency = [
     'dgraph4j' : 'io.dgraph:dgraph4j:21.03.1',
     'dropwizardMetricsCore': 'io.dropwizard.metrics:metrics-core:4.2.3',
     'dropwizardMetricsJmx': 'io.dropwizard.metrics:metrics-jmx:4.2.3',
-    'ebean': 'io.ebean:ebean:11.33.3',
-    'ebeanAgent': 'io.ebean:ebean-agent:11.27.1',
+    'ebean': 'io.ebean:ebean:12.16.1',
```
Comment (jjoyce0510) [inline build.gradle:64]: Just curious - why are these changes in here?

**Review by jjoyce0510 — APPROVED** [review]
LGTM, so long as it's clear why we are updating ebean (and what the implications will be)

File: build.gradle (hunk 1 above)
Comment (RyanHolstien) [reply build.gradle:64]: Ebean upgrade is due to a bug in the version that we are currently on that prevents troubleshooting any connection issues. They have a pretty glaring bug that results in every single error that happens at start-up to result in a NPE that masks the real error. Per the Ebean website, there are no necessary migration steps between 11 & 12 for us: https://ebean.io/docs/upgrading/

Comment (jjoyce0510) [reply build.gradle]: perfect - thank you. we've encountered this bug many times.

Comment (RyanHolstien) [reply build.gradle]: Looks like it caused a weird issue on this PR's tests even though on acryl-main it didn't so we rolled it back unfortunately :(

**Pull request merged**